crates/daphne-worker/src/aggregator/router/helper.rs

```diff
@@ -60,6 +61,12 @@ async fn agg_job(
 
         let hpke_receiver_configs = app.get_hpke_receiver_configs(req.version).await?;
 
+        let agg_param = DapAggregationParam::get_decoded_with_param(
+            &transition.task_config.vdaf,
+            &req.payload.agg_param,
+        )
+        .map_err(|e| DapAbort::from_codec_error(e, req.task_id))?;
+
```

Comment:
@mendess is this change required in `daphne-server` as well? AFAICT it doesn't need a custom implementation of this state machine.